README.md

```diff
@@ -1,3 +1,5 @@
+# LIGHTLY SSL
+
```

Comment:
```suggestion
```

Is there a particular reason why you would like the add the title? We already have the logo that includes the title:
![Screenshot 2024-10-07 at 11 40 40](https://github.com/user-attachments/assets/2b800743-7eef-4956-8599-262319ae12f9)

Reply:
ok i will reomve it